x-pack/plugins/lens/public/xy_visualization/xy_config_panel.tsx

```diff
@@ -98,10 +98,13 @@ export function LayerContextMenu(props: VisualizationLayerWidgetProps<State>) {
           defaultMessage: 'Chart type',
         })}
         name="chartType"
-        className="eui-displayInlineBlock"
+        className="eui-displayInlineBlock lnsLayerChartSwitch"
         options={visualizationTypes
           .filter((t) => isHorizontalSeries(t.id as SeriesType) === horizontalOnly)
           .map((t) => ({
+            className: `lnsLayerChartSwitch__item ${
+              layer.seriesType === t.id && 'lnsLayerChartSwitch__item-isSelected'
```

Comment:
This renders to `lnsLayerChartSwitch__item false` when the shortcircuit doesn't go through. A ternary operation may help: 

```ts
className: `lnsLayerChartSwitch__item ${
              layer.seriesType === t.id 
				? 'lnsLayerChartSwitch__item-isSelected' 
				: ''
			}`
```